<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite-1" duration-ms="1311" started-at="2020-10-25T17:17:44Z" finished-at="2020-10-25T17:17:45Z">
    <groups>
    </groups>
    <test name="SearchProperty" duration-ms="1311" started-at="2020-10-25T17:17:44Z" finished-at="2020-10-25T17:17:45Z">
      <class name="TestCases.LoginTestCases">
        <test-method status="PASS" signature="setUp()[pri:0, instance:TestCases.LoginTestCases@74a10858]" name="setUp" is-config="true" duration-ms="66811" started-at="2020-10-25T17:16:37Z" finished-at="2020-10-25T17:17:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="TEST_CASE_ONE()[pri:0, instance:TestCases.LoginTestCases@74a10858]" name="TEST_CASE_ONE" duration-ms="1118" started-at="2020-10-25T17:17:44Z" finished-at="2020-10-25T17:17:45Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at TestCases.LoginTestCases.TEST_CASE_ONE(LoginTestCases.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TEST_CASE_ONE -->
        <test-method status="PASS" signature="destroyDriver()[pri:0, instance:TestCases.LoginTestCases@74a10858]" name="destroyDriver" is-config="true" duration-ms="0" started-at="2020-10-25T17:17:50Z" finished-at="2020-10-25T17:17:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- destroyDriver -->
      </class> <!-- TestCases.LoginTestCases -->
    </test> <!-- SearchProperty -->
  </suite> <!-- Suite-1 -->
</testng-results>
